nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/RecordFieldType.java

```diff
@@ -265,6 +270,19 @@ public DataType getDataType(final String format) {
         return new DataType(this, format);
     }
 
+    /**
+     * Returns a Data Type that represents a "Decimal" with given precision
+     * @param precision number of digits to the right of point (.)
```

Comment:
The scale is actually the number of digits to the right of the point. The precision is the number of digits in the number.

Reply:
You right don't know what I was thinking when I wrote the comment, will fix that. thanks